docs/user-guides/s3_buckets.md

```diff
@@ -1,105 +1,37 @@
-|---|---|
-|List the contents of a bucket|`aws --no-sign-request --region <bucket's region> s3 ls s3://bucket_name/`|
-|List the contents of a directory in a bucket|`aws --no-sign-request --region <bucket's region> s3 ls s3://bucket_name/directory_name/`|
-|Download a file from a bucket|`aws --no-sign-request --region <bucket's region> s3 cp s3://bucket_name/directory_name/filename destination/path/filename`|
-|*Upload a file to a bucket|`aws --no-sign-request --region <bucket's region> s3 cp source/path/filename s3://bucket_name/destination/path/filename`|
-
-__*Increase your `s3.multipart_threshold` to allow uploading up to 5000MB as an anonymous user with the following steps:__
+*Prerequisite: an AWS access key that provides access to the S3 bucket*
 
 1. Open a terminal
-2. Run the following command:
-
-``` aws
-aws configure set default.s3.multipart_threshold 5000MB
-```
-    
-## **Accessing Private S3 Buckets**
----
-
-### **Configure the AWS Client in your OpenScienceLab account**
-
-(Prerequisite) To configure the AWS Client, you will need:
-- An _AWS Access Key ID_ to the account holding the bucket
-- An _AWS Secret Access Key_ to the account holding the bucket
-- An _arn_ to an _AWS IAM role_ with permission to access the bucket
-
-**Steps on configuring a terminal**:
-
-1. Open a terminal and run:
-```
-aws configure
-```
-
-2. It will ask you for the following:
+2. Run the command: `aws configure`
+3. When prompted, provide:
     - `AWS Access Key ID`
     - `AWS Secret Access Key`
-    - `Default region name`
-        - *Optional: You can enter the region where your bucket is located but this just sets a default and you will be able to enter your bucket's region in a following step*
-    - `Default output format`
-        - *Optional: you can leave this empty*
-            
-3. Manually edit the `config` file to add profile using *Vim.
+    - `Default region name`: "us-west-2"
+    - `Default output format`: "json" 
+
+  
+:::
 
-- Use `vim ~/.aws/config` to open the `config` file.
-- Add the following: 
-    
-```
-[profile osl]
-source_profile = default
-region = <your bucket's region>
-role_arn = <arn to your iam role>
-```
+:::{warning} 
 
-4. Save and exit Vim
+If accessing a public S3 bucket anonymously, you must either:
+- Provide the `--no-sign-request` argument to AWS-cli
+or
+- Provide an `UNSIGNED` config to the boto3 S3 bucket resource:
+    - `s3 = boto3.resource('s3', config=Config(signature_version=UNSIGNED))`
+ 
+:::
 
+## Example Notebook Demonstrating S3 Bucket Access with `AWS-cli` and `boto3`
 
-*NB: _Vim_ is a command line text editor. Since it does have a learning curve, users that never used Vim are encouraged to reference the [Vim Command Cheat Sheet](https://vim.rtorr.com/).
-    
-### **For Private S3 Bucket**
-_List of Useful AWS commands_:
+[S3_Buckets_Upload_Download.ipynb](https://github.com/ASFOpenSARlab/NISAR-Early-Adopters-Workshop-Oct-2024/blob/main/S3_Buckets_Upload_Download.ipynb)
 
```

Comment:
There is a weird empty block at the bottom:

![image](https://github.com/user-attachments/assets/54c0a7da-14fe-46c6-ae2a-364041f75f3b)

Reply:
There is a link there (currently hidden by JupyterLab/jupyter-myst link formatting bug. However, it points to an archived notebook, so I am pulling the section. We can put it back when we develop an introductory Jupyter Book for OSL that includes a similar example.